06_puppy/madhavansu/main_test.go

```diff
@@ -0,0 +1,207 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	tassert "github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+/**
+* Map store implemented in below scenarios
+* Dynamic Map ID: MapStore implementation of Storer backed by a map
+* Static Map ID: SyncStore implementation of Storer backed by a sync.Map
```

Comment:
This static/dynamic comment may change how you respond to my previous comments. However that means these comments belong on the structs you define and not here in the tests. Please move them if you retain the static/dynamic division.

Reply:
Updated.